tests/data/eltex-mes23xx_mes2348p.json

```diff
@@ -16785,7 +16785,7 @@
                     "active": 1,
                     "protocol": "lldp",
                     "remote_hostname": "bt-stadion-payload-ap-07",
-                    "remote_port": "bridge1/vlan75-payload (2cc81b5ba251)",
```

Comment:
Seems that interfaces are mixed up here

Reply:
let's wait for test to reach Eltex OS
now with reversed local_port_id find order